test/core/networking/protocol/authentication_test.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:test/test.dart';
+
+import 'package:flyde/core/networking/protocol/authentication.dart';
+import 'package:flyde/core/networking/protocol/protocol.dart';
+
+void main() {
+  group('AuthRequest', () {
+    test('In ProtocolDelegate.elements', () {
+      expect(ProtocolDelegate.elements['AuthRequest'], equals(isA<Deserialize>()));
```

Comment:
```suggestion
      expect(ProtocolDelegate.elements['AuthRequest'], isA<Deserialize>());
```

`equals` is not necessary here